samples/XAMLCatalog/XAMLCatalog/UIKitControls/UILabelViewController.h

```diff
@@ -17,5 +17,30 @@
 
 #import "MenuTableViewController.h"
 
-@interface UILabelViewController : MenuTableViewController
+@interface UILabelViewController : UIViewController <UITextFieldDelegate>
```

Comment:
agreed. now we are consistent.

---
In reply to: [105047481](https://github.com/Microsoft/WinObjC/pull/2178#discussion_r105047481) [](ancestors = 105047481,104795774)